### PR TITLE
EOS-22259 Populate DIX Pool layout parameters to confstore file

### PR DESCRIPTION
**Pull request by zahidsk**
Signed-off-by: zahid.shaikh <zahid.shaikh@seagate.com>

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: lr-cli/cortx_setup/commands/storageset/config/durability.py [hunk 1]
```diff
@@ -76,34 +74,35 @@ def run(self, **kwargs):
                 f'json://{CONFSTORE_CLUSTER_FILE}'
             )
 
-            ss_name = Conf.get(index, f'cluster>{cluster_id}>storage_set[0]>name')
-
-            durability_dict = {durability_type: {}}
+            ss_name = Conf.get(index,
```
Comment (nitin-seagate) [inline lr-cli/cortx_setup/commands/storageset/config/durability.py:77]: How will this work for multiple storage sets?
Tomorrow we may have 2-3 storage sets in which case only storage_set[0] will not work

**Review by nitin-seagate — CHANGES_REQUESTED** [review]

File: lr-cli/cortx_setup/commands/storageset/config/durability.py (hunk 1 above)
Comment (nitin-seagate) [reply lr-cli/cortx_setup/commands/storageset/config/durability.py:77]: Lets leave storage-set for now

**Review by nitin-seagate — CHANGES_REQUESTED** [review]
Not sure this changes are correct

**Review by nitin-seagate — APPROVED** [review]
Looks good to me

**Pull request merged**